src/otx/core/model/visual_prompting.py

```diff
@@ -916,6 +974,9 @@ def _customize_inputs(  # type: ignore[override]
         """Customize OTX input batch data entity."""
         images: list[np.ndarray] = []
         processed_prompts: list[dict[str, Any]] = []
+        print(entity.labels)
+        print(entity.prompts)
+        print(entity.polygons)
```

Comment:
Is there a reason to output this customize_inputs every time?

Reply:
The PR is still WIP, to be deleted